### ISSUE TITLE
exporting a footage crashes OpenShort Video Editor 2.5.1 (2020/04/01)

### ISSUE THREAD

**Issue by gr-hayduk** [OpenShot/openshot-qt#3338]
**Describe the bug**
exporting crashes

**Steps to reproduce the behavior:**
1. Load a video project with some media source (I have a MOV file from my Canon EOS 650D)
2. Export it with Advanced parameters:
Advanced Options - Video & Autdio, Start Frame: 1 End Frame: 1000 (shorten it, don't export whole video!)
Profile - leave as it is
Video Settings - chanage Bit Rate to 10 Mb/s (mp4, libx264)
Audio Settings - aac, 44100, Mono, 128 kb/s
GET BACK to Advanced options, End Frame AGAIN needs to be shortened to 1000 frames (gets overwritten with full number of frames)

**Expected behavior**
A mp4 file should be created, OpenShot should'nt disappear (get killed? crash?).

**System Details**
- OpenShot Version: 2.5.1-dev2
- libopenshot Version:  0.2.5-dev2
- Platform: Windows-10-10.0.18362
- Processor: Intel64 Family 6 Model 58 Stepping 9, GenuineIntel
- Machine: AMD64
- Python version: 3.7.4
- Qt5 version: 5.12.4
- PyQt5 version: 5.12.3
- Qt Detected Languages: ['pl-PL', 'en-US']
- LANG Environment Variable: 
- LOCALE Environment Variable: 
- Daily Build Installed: http://github.com/OpenShot/openshot-qt/releases/download/OpenShot-v2.5.1-dev2-1585351843-166f6938-232f3f79-x86_64.exe

**Log Files**
- http://logs.openshot.org/wgmc3ll863fwbwndy1ury94lygaes4w1/openshot-qt.zip
- http://logs.openshot.org/wgmc3ll863fwbwndy1ury94lygaes4w1/libopenshot.zip

**Exception / Stacktrace**
```No stacktrace found in log files```

**Screenshots** (Optional)
OpenShort disappears, there's nothing to take screenshot.

**Comment by SuslikV** [comment]
I cannot reproduce this. Is this only the project you have issues with?

**Comment by rexdk** [comment]
This looks like:
Exporting a video with audio set to mono triggers an OpenShot crash #3361

**Comment by SuslikV** [comment]
Maybe it source file specific? I tried import, export both import&export as mono audio. And was unable to reproduce.

**Comment by rexdk** [comment]
I was experimenting with 'clip properties', specifically 'volume mxing'. I had a short mp3 loaded onto 2 tracks (ie as 2 clips), and a single png on another track. No video. Then I exported to mp4 with audio set to mono. It happened even if the volume mixing was unchanged (set to None). 

Might it be Win10 related?

I could try an even more minimal case if that would help.

A related issue may be that (I think) when you try to export as 5.1, the resulting file does not show as 5.1 when it is imported back into OpenShot - it shows as 'unspecified', although there are 6 channels. . I've not fully explored this, so have not raised it before. And I think 5.1 may not yet be part of the mp4 spec so this could be  perfectly OK - and  I don't know what the text description is supposed to do. Just trying to offer an idea.

**Comment by SuslikV** [comment]
@rexdk .mp3 files may has cover image, thus you may run into different issue.

**Comment by rexdk** [comment]
@SuslikV Thanks - but...... I'm afraid I don't understand that :)

**Comment by SuslikV** [comment]
@rexdk try to remove the cover images from audio as described in https://github.com/OpenShot/openshot-qt/issues/2756#issuecomment-487876270 

Do you see any crash message from the OS itself when it happens? How it looks, your crash?

**Comment by rexdk** [comment]
@SuslikV Thanks, sorry for the delay.
There is no cover art; the mp3 was generated by Audacity - so I think this is not that issue.

To test further I have downloaded a Big Buck Bunny mp4 video with stereo, and tried with that:
1.  Open OpenShot, import mp4, load onto track, drag righthand edge to shorten the movie, export>advanced>audio>mono.
   The export starts, but after about 6%, OpenShot disappears from the screen (no error message).
2.  Open OpenShot, import mp4, load onto track, drag righthand edge to shorten the movie, Channel Filter=0, Channel mapping=0 (so it's a mono audio stream), export>advanced>audio>mono.
   The export starts, but after about 15%, OpenShot disappears from the screen.

I've saved log files from 1.  I will post if it helps and if you tell me which file will help

Should this now go into #3361 ?
